jekyll/_cci2/getting-started.md

```diff
@@ -169,8 +150,8 @@ and copy and paste the following text into it.
    ```
 
 
-1. Commit these changes to your repository and navigate back over to the
-    CircleCI Pipelines page. You should see your CircleCI pipeline running.
+1. Commit these changes to your repository and navigate back to the
+    CircleCI Pipelines page. You should see your pipeline running.
```

Comment:
I'm going to just mark this last example I see, but we may want to go back and bold the pages and tabs we reference in the UI since that's something we've been doing to indicate the user needs to be in the UI. Like **Pipelines** page / **Projects** page.